_posts/2021-09-20-logicdrop-customer-story.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: 'How Quarkus Improved Logicdrop's Developer Experience and Process'
```

Comment:
```suggestion
title: "How Quarkus Improved Logicdrop's Developer Experience and Process"
```

Reply:
Fixed.  As a note, i copied the previous article here to use as a template.

Reply:
Fixed